nasa/cmr-metadata-review
# CMRARC-866: CMR-Dashboard Vulnerabilities June 2024

# Overview

### What is the feature?

Fixing vulnerabilities related to actionpack and braces

### What is the Solution?

updated actionpack by adding a gem to the gemfile and updated braces by regenerating yarn.lock

### What areas of the application does this impact?

gemfile/lock and yarn.lock

# Testing

### Reproduction steps

- **Environment for testing: Local
- **Collection to test with: N/A

1. run $ yarn audit for package.json
2. run $ bundle-audit check --update for gemfile

### Attachments

N/A

# Checklist

- [x ] I have performed a self-review of my own code
- [ x] My changes generate no new warnings
